test/test_linharn.py

```diff
@@ -97,10 +106,11 @@ def proc():
 
 @fixture
 def linux_harn(scope="function"):
-  process = Process(target=wrap_loop)
-  yield process
+  proc_list = Linharn_proc()
+  yield proc_list
   try:
-      process.terminate()
+      for proc in proc_list.procs:
+        proc.terminate()
```

Comment:
should the try/except be around this terminate command?

right now if one terminate throws, all the remaining may not be terminated.